### PR TITLE
Bugfix/repair ingress mechanism

### PR DESCRIPTION
**Pull request by deprecastor**

### CODE REVIEW

File: charts/vaultwarden/README.md [hunk 1]
```diff
@@ -45,19 +45,20 @@ $ helm install some-vaultwarden edudip/vaultwarden
...
-| `service.sessionAffinity`       | Service session afffinity                   | `""`    |
-| `service.sessionAffinityConfig` | Additional settings for the sessionAffinity | `{}`    |
+| Name                            | Description                                                           | Value   |
+| ------------------------------- | --------------------------------------------------------------------- | ------- |
+| `ingress.enabled`               | turn on/off ingress of the chart at all                               | `false` |
+| `ingress.className`             | Name of the Ingress Class                                             | `""`    |
+| `ingress.annotations`           | Annotations to add to the ingress object                              | `{}`    |
+| `ingress.hosts`                 | Array of host objects                                                 | `[]`    |
+| `ingress.tls`                   | Array of TLS configurations                                           | `[]`    |
+| `ingress.servicePort`           | Port number of the service to reach HTTP endpoint defaults to prot 80 | `""`    |
```
Comment (mariusziemke) [inline charts/vaultwarden/README.md:55]: Typo `port` instead of `prot`.

File: charts/vaultwarden/values.yaml [hunk 2]
```diff
@@ -137,6 +137,7 @@ affinity: {}
 ## @param ingress.annotations [object] Annotations to add to the ingress object
 ## @param ingress.hosts [array] Array of host objects
 ## @param ingress.tls [array] Array of TLS configurations
+## @param ingress.servicePort [string] Port number of the service to reach HTTP endpoint defaults to prot 80
```
Comment (mariusziemke) [inline charts/vaultwarden/values.yaml:140]: Same typo with `prot`

File: charts/vaultwarden/values.yaml [hunk 3]
```diff
@@ -155,6 +156,7 @@ ingress:
   ##    hosts:
   ##      - chart-example.local
   ##
+  servicePort: ""
```
Comment (mariusziemke) [inline charts/vaultwarden/values.yaml:159]: Why don't you put 80 in here and get rid of the default?

**Review by mariusziemke — CHANGES_REQUESTED** [review]

File: charts/vaultwarden/README.md (hunk 1 above)
Comment (deprecastor) [reply charts/vaultwarden/README.md]: will be solved when auto generation is working again

**Review by dennishoffmann-edu — APPROVED** [review]

**Review by aborrmann — APPROVED** [review]

File: .github/workflows/generateReadme.yaml [hunk 4]
```diff
@@ -20,9 +20,11 @@ jobs:
       - name: "Install Node"
         uses: actions/setup-node@v3
         with:
-          node-version: 16
+          node-version: '16.x'
```
Comment (mariusziemke) [inline .github/workflows/generateReadme.yaml:23]: why did you change that?

**Review by mariusziemke — APPROVED** [review]
